src/System.Reflection.MetadataLoadContext/tests/src/Tests/TypeInfoFromProjectN/TypeInfo_PropertyTests.cs

```diff
@@ -81,7 +81,7 @@ public static void TestGuid()
             TypeInfo ti = t.GetTypeInfo();
 
             Guid myguid = ti.GUID;
-            Assert.NotNull(myguid);
+            Assert.NotEqual(myguid, Guid.Empty);
```

Comment:
The arguments here should be swapped.  The first is expected, the second is actual.

Reply:
I can't believe I did that again! I'll resolve shortly.